tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -52,33 +61,59 @@ def signin(self, username: str, password: str):
         )
 
         if response.status_code == 503:
-            print(f"Server url {self.url} not available.")
+            print(f"Server '{self.url}' not available.")
             self.session.close()
             sys.exit()
 
-        response_json: dict = response.json()
-
-        status: str = response_json['data']['signin']['status']
-        message: str = response_json['data']['signin']['message']
-
-        if status == 'SUCCESS':
-            log.debug(f"Success: Signed into {self.url} as {username}.")
+        response_json: dict = response.json().get('data', {}).get('signin', {})
+        
+        if response_json.get('status') == 'SUCCESS':
+            message = f"Success: Signed into {self.url} as {username}." 
+            log.info(message)
+            print(message)
             self.signin_status = 'success'
-        elif status == 'FAILED':
+        elif response_json.get('status') == 'FAILED':
+            message = f"Error: Unable to sign into {self.url} as {username}.\n{response_json.get('message')}"
             log.error(message)
+            print(message)
             self.signin_status = 'failed'
         else:
-            log.error('Error: sign in failed, exiting.')
+            log.error(f"Error: sign in failed, exiting. Sign in status: {response_json.get('status')}")
+            print(f"Unable to sign into {self.url} as {username}")
             self.signin_status = 'failed'
 
-    def list_databases(self):
+        self.username = username
+        self.schemas = self.list_schemas()
+
+    def sign_out(self):
+        """Signs the client out of the EMX2 server."""
+        response = self.session.post(
+            url=self.api_graphql,
+            json={'query': queries.sign_out()}
```

Comment:
interesting to move the graphql to another file in this way so thing keep readable. I clearly need to learn python tricks.